CHANGELOG.md

```diff
@@ -3,6 +3,7 @@ Changes in this release:
 - Add support to `LsmProject.compile` to have multiple instances selected
 - Add `LoadGenerator` helper to generate some load on the remote orchestrator
 - Add `--lsm-dump-on-failure` option, allowing to generate a support archive of the orchestrator when a test fails, and save it in the host /tmp directory. (#409)
+- Fix race condition in `RemoteServiceInstance.wait_for_state` that would make it return a `ServiceInstance` with a different version that the one we asked the method to wait for.
```

Comment:
Perhaps be a bit more explicit (and less alarming)?
```suggestion
- Fix race condition in `RemoteServiceInstance.wait_for_state` that would make it return a `ServiceInstance` for the latest version rather than the one we asked the method to wait for.
```